### DISCUSSION TITLE
I got a Error

### DISCUSSION THREAD

**Discussion by bigfranko** [DIGITALCRIMINALS/OnlyFans#520] [Q&A]
Ok so I got this event loop closed error. Any ideas?
<img width="939" alt="ScreenShot_20220723154700" src="https://user-images.githubusercontent.com/66663872/180622545-4b43ff81-4a97-4dad-9b44-af785e2d9a24.png">

**Comment by Egon099** [comment]
https://github.com/DIGITALCRIMINALS/OnlyFans/issues/518#issuecomment-1193026208
Seems same error like I got. Since I use latest this helped me. Maybe it helps you.

**Reply by AnotherSyntaxError** [reply]
Easy way to update without losing settings? I'm a real python noob.

**Reply by Egon099** [reply]
I didn't loose settings since it doesn't effect the settings file. It is simple changing the file text. I don't know nothing about python or programming and managed to do it.
finding the right line was easy too with notepad++
That is why I linked right to the solution.
